### PR TITLE
adding angular source map files

### PR DESCRIPTION
**Pull request by jstenhouse**
@holtbp @rantler 

Gets rid of console errors.

### CODE REVIEW

File: vendor/assets/javascripts/angular-route.min.js.map [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...onNgViewEnter","autoScrollExp","$eval","$emit","onloadExp","autoscroll","onload","$on","ngViewFillContentFactory","$compile","$controller","html","contents","controller","$scope","controllerAs","data","children","ngRouteModule","module","provider","$RouteProvider","inherit","parent","extra","extend","pathRegExp","path","opts","insensitive","caseInsensitiveMatch","ret","keys","replace","_","slash","key","option","optional","star","push","regexp","RegExp","routes","when","this.when","route","redirectPath","length","substr","otherwise","this.otherwise","params","$get","$rootScope","$location","$routeParams","$q","$injector","$http","$templateCache","$sce","updateRoute","next","parseRoute","last","$$route","equals","pathParams","reloadOnSearch","forceReload","copy","$broadcast","redirectTo","isString","interpolate","search","url","then","resolve","template","templateUrl","forEach","value","get","invoke","isFunction","getTrustedResourceUrl","loadedTemplateUrl","response","all","error","match","m","exec","on","i","len","val","decodeURIComponent","name","string","result","split","segment","segmentMatch","join","reload","$evalAsync","$RouteParamsProvider","this.$get","directive","$inject"]
```
Comment (briandela) [inline vendor/assets/javascripts/angular-route.min.js.map:7]: No indentation?

**Comment by briandela** [comment]
Looks like vendor/assets/javascripts/angular.min.js.map isn't actually changed - at least github doesnt think so

**Comment by jstenhouse** [comment]
I added both files new as downloaded from angular's site. Source map files are nice for debugging in the browser. At least that's what I read.

File: vendor/assets/javascripts/angular-route.min.js.map (hunk 1 above)
Comment (jstenhouse) [inline vendor/assets/javascripts/angular-route.min.js.map:7]: This is how the file came from angular. I didn't want to edit/mess with it.

**Comment by briandela** [comment]
Ah, the github diff tool said the file was blank - I didn't actually pull down the branch to check.. Indeed, anuglar is correct, codemap files are awesome - they allow the debugger in tools like chrome dev tools to debug your JS easier.

Merge away good sir, merge away.

**Pull request merged**